packages/grid/enclave/attestation/start.sh

```diff
@@ -5,7 +5,14 @@ export PATH="/root/.local/bin:${PATH}"
 APP_MODULE=server.attestation_main:app
 APP_LOG_LEVEL=${APP_LOG_LEVEL:-info}
 UVICORN_LOG_LEVEL=${UVICORN_LOG_LEVEL:-info}
-HOST=${HOST:-0.0.0.0}
+if grep -q ":" /proc/net/if_inet6 ; then
```

Comment:
Is it possible we can make this configurable to allow both IPV4, IPV6 and both if desired?
https://github.com/encode/uvicorn/discussions/1529

I think a few env variables would be helpful for outside configuration.

Reply:
I usually use hypercorn, which supports dual stack.